tests/nvim-autopairs_spec.lua

```diff
@@ -169,6 +169,36 @@ local data = {
         before = [[("abcd|}} ]],
         after  = [[("abcd}|} ]]
     },
+    {
+        -- ref: issue #331
+        name = "do not move right on non-end-pair char" ,
+        setup_func = function()
+          npairs.add_rule(Rule("$","$"):with_move(cond.done()))
+        end,
+        key    = [[(]],
+        before = [[$|$]],
+        after  = [[$(|)$]]
```

Comment:
This test does not pass, I don't understand why..

Adding some debugs, it seems the `can_pair` call of `(` returns `false` in this case, but looking at the code I don't understand why :thinking:

Reply:
That finally explains it, thanks! I used another symbol for that test.